agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -713,3 +713,40 @@ func TestNetworkModeBridge(t *testing.T) {
 		t.Fatalf("Networking mode bridge testing failed, err: %v", err)
 	}
 }
+
+// TestDockerConcurrentPull tests the agent can perform concurrent pull for docker >= 1.9.0
+// Start 4 tasks totally 40 containers at the same time.
```

Comment:
I think you mean "totaling" instead of "totally".  Also, I think your comment is out-dated with respect to which version of Docker is the minimum here.